docs/schedule.md

```diff
@@ -1,20 +1,21 @@
 # OSGVS21 Schedule
 
-This page contains schedule details for **full, invited participants** of the OSG Virtual School 2021.
+This page contains schedule details for **full, invited participants** of the OSG Virtual School 2021,
+who were selected following a Spring 2021 application process.
```

Comment:
Except that this is not accurate for the 2020 applicants.

Reply:
fixed